tests/backstop_data/backstop.tpl.json

```diff
@@ -16,28 +16,68 @@
   },
   "scenarios": [
     {
-      "label": "Mosaico - Main Page",
-      "url": "{url}/civicrm/a/#/mosaico-template",
-      "onReadyScript": "mosaico/main-page.js"
+      "label": "Mosaico - Mailing list - Step 1",
+      "url": "{url}/civicrm/mailing/browse/unscheduled?reset=1&scheduled=false",
+      "onReadyScript": "mosaico/mailing/step-1.js"
     },
     {
-      "label": "Mosaico - Main Page - hover state",
-      "url": "{url}/civicrm/a/#/mosaico-template",
-      "onReadyScript": "mosaico/main-page-hover-state.js"
+      "label": "Mosaico - Mailing list - Delete draft",
```

Comment:
sounds like you are deleting something. Shouldn't the title be something like "Delete draft modal"